doc/source/serve/production.md

```diff
@@ -144,7 +141,7 @@ fruit_config.yaml
 The `fruit_config.yaml` file contains:
 
 ```yaml
```

Comment:
Can we remove host and port from this yaml example? They're not included in the 2.0 release.